refactor: minor fixes on kubeadm deployment scripts (#44)
aneoconsulting/ArmoniK.Infra: kubernetes/onpremise/kubeadm/examples/complete/README.md

<!-- BEGIN_TF_DOCS -->
## Requirements

| Name | Version |
|------|---------|
| <a name="requirement_terraform"></a> [terraform](#requirement\_terraform) | >= 1.0 |
| <a name="requirement_null"></a> [null](#requirement\_null) | >= 3.2.1 |
| <a name="requirement_random"></a> [random](#requirement\_random) | ~> 3.5.1 |

## Providers

| Name | Version |
|------|---------|
| <a name="provider_null"></a> [null](#provider\_null) | >= 3.2.1 |
| <a name="provider_random"></a> [random](#provider\_random) | ~> 3.5.1 |

## Modules

| Name | Source | Version |
|------|--------|---------|

> | <a name="module_install_kubeadm_cluster"></a> [install\_kubeadm\_cluster](#module\_install\_kubeadm\_cluster) | ../.. | n/a |

Reviewer: add the name of the module

> | <a name="module_install_kubeadm_cluster"></a> [install\_kubeadm\_cluster](#module\_install\_kubeadm\_cluster) | ../.. | n/a |

Reviewer: ```suggestion
| <a name="module_install_kubeadm_cluster"></a> [install\_kubeadm\_cluster](#module\_install\_kubeadm\_cluster) | ../../kubeadm | n/a |
```

> # OnPremise kubeadm

Reviewer: ```suggestion
# Onpremise Kubeadm
```

> ### how to use it : 
> You have to update values PUBLIC_DNS_HERE, PRIVATE_DNS_HERE and TLS_PRIVATE_KEY_FILE_HERE
> in the file [terraform.tfvars](terraform.tfvars)

Reviewer: ```suggestion
The Terraform scripts use the module [kubeadm](../../kubeadm) to install and configure a Kubernetes cluster with `Kubeadm`.

First, you should have a list of barre metal ou virtual machines, and then you have to update values `PUBLIC_DNS_HERE`, `PRIVATE_DNS_HERE` and `TLS_PRIVATE_KEY_FILE_HERE` in the file [terraform.tfvars](terraform.tfvars).
```

| Name | Version |
|------|---------|

> | <a name="provider_null"></a> [null](#provider\_null) | n/a |

Reviewer: Add minimal version

Author: ok
